src/vea/tests/vea_ut.c

```diff
@@ -284,10 +284,6 @@ ut_reserve_bitmap(void **state)
 	assert_rc_equal(rc, 0);
 
```

Comment:
Looks vea_format() is called in this sub test, I suppose ut_setup() should be called as well?